drive/activity-v2/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
```

Comment:
Can you add a license header and region start/end tag?
```html
<!--
Copyright 2018 Google LLC
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    https://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->
<!-- [START drive_activity_v2_quickstart] -->
...
<!-- [END drive_activity_v2_quickstart] -->
```


https://github.com/gsuitedevs/browser-samples/blob/master/drive/activity/index.html#L13

Reply:
Done.